src/modules/settings/AppSettings.js

```diff
@@ -26,7 +23,8 @@ const e2eSettings = {
   introShown: true,
   errorTracking: false,
   allowPushNotifications: false,
-  proposeNearbyCities: false
+  proposeNearbyCities: false,
+  apiUrlOverride: null
```

Comment:
That's actuelly pretty neat here :) Can you set here "https://cms-test.integreat-app.de" ? That way the end2end tests do not affect the live system.